cmd/kubeadm/app/cmd/config.go

```diff
@@ -401,7 +404,7 @@ func AddImagesListFlags(flagSet *flag.FlagSet, cfgPath *string) {
 	flagSet.StringVar(cfgPath, "config", *cfgPath, "Path to kubeadm config file.")
```

Comment:
Can you please move this to `AddImagesCommonConfigFlags`? That will solve the issue

Reply:
little confused about it. `AddImagesCommonConfigFlags` are common flag for 'list' and 'pull' flag, why move it?

Reply:
`--config` needs to be common between both `list` and `pull`. Both use the config file.
The thing that pull needs that list don't is `CRISocket`, so that should be passed correctly (!) to pull only
Makes sense?

Reply:
got it, thanks